### ISSUE TITLE
Add Feature-Policy header

### ISSUE THREAD

**Issue by comassky** [containous/traefik#4936]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

A new security response header is released : Feature-Policy 

> The HTTP Feature-Policy header provides a mechanism to allow and deny the use of browser features in its own frame, and in content within any <iframe> elements in the document.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy

Is it possible to add it like : 

`traefik.frontend.headers.FeaturePolicy=VALUE`

(for exemple)

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of communication regulation.

Thanks for your interest in Traefik!

Issue templates help us help you by providing all necessary information.

Please edit your issue and use the available templates:

- [Feature request](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Feature_request.md)
- [Bug report](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Bug_report.md)

And remember: each time someone ignores the template, a cute little bunny dies.
<details>
<summary>#SaveTheCuteBunny :heart: :rabbit: :heart:</summary>

![southpark - estrella kill rabbit](https://user-images.githubusercontent.com/5674651/27143914-3cea7896-5130-11e7-81f8-02d8ac2da7a7.gif)

![southpark - estrella kill rabbit2](https://user-images.githubusercontent.com/5674651/27143924-49104c72-5130-11e7-82ae-ccedbd5dfa1c.gif)

</details>

**Comment by dduportal** [comment]
Hi @comassky , is this header working when using `traefik.frontend.headers.customRequestHeaders` (ref. https://docs.traefik.io/configuration/backends/docker/#custom-headers)?

**Comment by guillaumedsde** [comment]
> Hi @comassky , is this header working when using `traefik.frontend.headers.customRequestHeaders` (ref. https://docs.traefik.io/configuration/backends/docker/#custom-headers)?

Hi, I just tested it by adding the following label to  my traefik container:
```yaml
traefik.frontend.headers.customResponseHeaders: "Feature-Policy:camera 'none'; fullscreen 'none'; geolocation 'none'; microphone 'none'; payment 'none'; speaker 'none'; usb 'none'; vibrate 'none'; vr 'none';"
```

and tested it with [securityheaders.io](https://securityheaders.com/) and I can confirm it works

**Comment by comassky** [comment]
Thanks, it's KO for me ...

CustomHeaders add the Feature policy, but remove all the others Headers (like traefik.frontend.headers.contentSecurityPolicy etc etc).

Best option should be to add a parameter with a **traefik.frontend.headers.FeaturePolicy**

**Comment by guillaumedsde** [comment]
> Thanks, it's KO for me ...
> 
> CustomHeaders add the Feature policy, but remove all the others Headers (like traefik.frontend.headers.contentSecurityPolicy etc etc).
> 
> Best option should be to add a parameter with a **traefik.frontend.headers.FeaturePolicy**

+1 for adding the header, but don't remove, [Content Secutiry Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), its a different header from Feature Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy)

**Comment by dduportal** [comment]
Thanks for your feedbacks. The issue has been prioritized as a new feature (more information on https://docs.traefik.io/v2.0/contributing/maintainers/#labels if you are curious).

No ETA, has the priority is the `v2.0` is currently worked on by maintainers.

**Comment by traefiker** [comment]
Closed by #5156.
